### PR TITLE
Transitioning Icon and HelpTooltip to reference latitude/colors

### PR DESCRIPTION
**Pull request by FermiDirak**
This PR transitions HelpTooltip and Icon to reference latitude/colors for iconColors.

Monorepo PR:
https://github.flexport.io/flexport/flexport/pull/53289

### CODE REVIEW

File: colors.js [hunk 1]
```diff
@@ -121,3 +122,51 @@ export const v3ColorMap: ColorMap = Object.freeze({
   purple50: latitudeColors.purple50,
   purple60: latitudeColors.purple60,
 });
+
+export const v3ColorKeyMap: {[oldColor: Color]: LatitudeColor} = Object.freeze({
```
Comment (ddzoan) [inline colors.js]: What is this going to be used for?

**Comment by dmnd** [comment]
I'd consider calling `LatitudeColor` just `Color` since that type is going to end up in the API. But that's a different PR.

File: styles/getThemeColors.js [hunk 2]
```diff
@@ -6,11 +6,10 @@
 
 import type {Theme} from "../context/ThemeNameContext";
 import themeColorMaps from "./themeColorMaps";
-import {type Color} from "./colors";
 
 export type ThemeColors = {|
-  +primary: Color,
-  +brand: Color,
+  +primary: "blue30" | "green40",
+  +brand: "blue40" | "green30",
```
Comment (Cytancy) [inline styles/getThemeColors.js]: Could this be:
```
export type ThemeColorMap = {|
  +primary: "blue30",
  +brand: "blue40",
|} | {|
  +primary: "green40",
  +brand: "green30",
|};
```
For `BASE` + `TRANSMISSION` and it doesn't look like it allows pairings outside these two.

Comment (FermiDirak) [reply styles/getThemeColors.js]: good call. Changes made 👍

File: colors.js (hunk 1 above)
Comment (FermiDirak) [reply colors.js]: Currently this is used to shim one difficult to convert usage of styles/colors: https://github.flexport.io/flexport/flexport/pull/53289/files#diff-c3707e2f07eb1623a8a998bc7189bdc6R14

**Comment by FermiDirak** [comment]
I figured out how I can transition from styles/colors to latitude/colors without any intermediary steps. Closing this PR in favor of https://github.com/flexport/latitude/pull/55

**Pull request closed without merging**